data-connect/flutter-blank/proxy-web/proxy.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+const EXT_PORT = 9002;
```

Comment:
Can these look for a ENV and fallback to these values if not set? To help with updating in the future